README.md

```diff
@@ -22,21 +22,21 @@ You have been provided a [JSON object](js/index.js) with all the necessary data
 
 * [ ] Submit a Pull-Request to merge <firstName-lastName> Branch into master (student's  Repo). **Please don't merge your own pull request**
 * [ ] Add your project manager as a reviewer on the pull-request
-* [ ] Your project manager will count the project as complete by merging the branch back into master.
+* [ ] Your project manager will count the project as complete by merging the branch back into master. -->
 
 ## Task 1: Create selectors to point your data into elements
-* [ ] Create selectors by using any of the DOM element's methods
-* [ ] Note that IDs have been used on all images.  Use the IDs to update src path content
+* [x] Create selectors by using any of the DOM element's methods
```

Comment:
The fact that you tick the boxes says alot about how much effort your putting in, nice job man